tests/test_query.py

```diff
@@ -39,6 +39,7 @@ def _check_graph_makes_sense(g, node_feature_names, edge_feature_names):
     os.close(f)
 
     try:
+        g.targets[targets.BINARY] = 0
```

Comment:
because `_check_graph_makes_sense` defines a GraphDataset instance that is in training mode, and now this requires a target to be defined (otherwise you can't train a model)